homeassistant/components/vera.py

```diff
@@ -164,7 +164,7 @@ def device_state_attributes(self):
         attr = {}
 
         if self.vera_device.has_battery:
-            attr[ATTR_BATTERY_LEVEL] = self.vera_device.battery_level + '%'
+            attr[ATTR_BATTERY_LEVEL] = self.vera_device.battery_level
```

Comment:
@balloob you might be right on that.

The original intention was for InfluxDB, which would handle the string to number conversion.

Should we wrap `self.vera_device.battery_level` around a `float()`?